CHANGELOG.md

```diff
@@ -1,5 +1,126 @@
 # Changelog
 
+## 0.6.0
+
+### Breaking changes
+
+See also, the "Remapped rules" section which may result in disabled rules.
+
+- The pytest rules `PT001` and `PT023` now default to omitting the decorator parentheses when there are no arguments ([#12838](https://github.com/astral-sh/ruff/pull/12838)).
+
+- Detect imports in `src` layouts by default ([#12848](https://github.com/astral-sh/ruff/pull/12848))
+
+- Enable Jupyter Notebook formatting and linting by default ([#12878](https://github.com/astral-sh/ruff/pull/12878)).
+
+    You can disable specific rules for notebooks using [`per-file-ignores`](https://docs.astral.sh/ruff/settings/#lint_per-file-ignores):
+
+    ```toml
+    [tool.ruff.lint.per-file-ignores]
+    "*.ipynb" = ["E501"] # disable line-too-long in notebooks
+    ```
+
+    If you'd prefer to either only lint or only format Jupyter Notebook files, you can use the
+    section-specific `exclude` option to do so. For example, the following would only lint Jupyter
+    Notebook files and not format them:
+
+    ```toml
+    [tool.ruff.format]
+    exclude = ["*.ipynb"]
+    ```
+
+    And, conversely, the following would only format Jupyter Notebook files and not lint them:
+
+    ```toml
+    [tool.ruff.lint]
+    exclude = ["*.ipynb"]
+    ```
+
+    You can completely disable Jupyter Notebook support by updating the [`extend-exclude`](https://docs.astral.sh/ruff/rules/superfluous-else-continue/) setting:
+
+    ```toml
+    [tool.ruff]
+    extend-exclude = ["*.ipynb"]
+    ```
+
+### Deprecations
+
+The following rules are now deprecated:
+
+- [`pytest-missing-fixture-name-underscore`](https://docs.astral.sh/ruff/rules/pytest-missing-fixture-name-underscore/) (`PT004`)
+- [`pytest-incorrect-fixture-name-underscore`](https://docs.astral.sh/ruff/rules/pytest-incorrect-fixture-name-underscore/) (`PT005`)
+- [`unpacked-list-comprehension`](https://docs.astral.sh/ruff/rules/unpacked-list-comprehension/) (`UP027`)
+
+### Remapped rules
+
+The following rules have been remapped to new rule codes:
+
+- [`unnecessary-dict-comprehension-for-iterable`](https://docs.astral.sh/ruff/rules/unnecessary-dict-comprehension-for-iterable/): `RUF025` to `C420`
```

Comment:
I would put these two sections above the first bullet point you included in the "Breaking changes" section -- and maybe also above the section about Jupyter notebooks now being linted/formatted by default. I think they're more breaking than the change to the default for `PT001` and `PT023`, since these changes might require you to make changes to your Ruff config file to keep your CI green and free of warnings.

Reply:
I prefer keeping it as is because it is more consistent with past releases. We can shorten the first bullet points by removing the lengthy Jupyter Notebook section and instead link to the configuration (and blog post). We can keep the lengthy explanation in the `BREAKING_CHANGES.md`

Reply:
Yes, shortening the Jupyter notebook section might be good. I would also at least swap the order of the two bullets you have in the "Breaking changes" section

Reply:
Can you expand on which bullets you would swap? Is it the `src` change and the change of the rule defaults?

Reply:
Ah, sorry! I suppose I would probably put the (shortened) Jupyter notebook bullet first, then the `isort` bullet, and the bullet about the `PT` rules last.